add Escrow and StableToken to the Registry

yerdua opened the pull request:
### Description

ContractKit will need to be able to get addresses for contracts from the Registry. Escrow and StableToken were missing. This commit modifies the deployment of those contracts to add them to the registry.

### Tested
Steps:
1. Deployed a testnet (`yerdua`) with these contract changes
2. Switched to the WIP branch for #287 
3. Built contractkit for that testnet: `yarn build yerdua`
4. Ran a test that logged the addresses as retrieved from the registry along with the actual addresses, and got this output, which contains the addresses for Escrow and StableToken:
```
Using the registry
    getRegistryContract: Address of Registry contract is 0x000000000000000000000000000000000000ce10
    getRegistryContract: Address of Attestations contract is 0xAfDF1963Dac816dd84B49B8d2179296Ab32D6519
    getRegistryContract: Address of Escrow contract is 0x107d9410739e1F557BAD3b0CfA955833070aBaAe
    getRegistryContract: Address of Exchange contract is 0x95b95256D906664c5047d50Fd2F34cAb741CB44D
    getRegistryContract: Address of GasPriceMinimum contract is 0x63362e42BF791871ab570AD60B4228C02d9a68F3
    getRegistryContract: Address of GoldToken contract is 0xbe7E2bf1d46f7F351a1785e707Eae8f252Af94Dd
    getRegistryContract: Address of StableToken contract is 0x4917775b54738BFf899C91f320d260Be413f1d16

Reading the addresses from the contract instances:
    getRegistryContract2: Address of Registry contract is 0x000000000000000000000000000000000000ce10
    getRegistryContract2: Address of Attestations contract is 0xAfDF1963Dac816dd84B49B8d2179296Ab32D6519
    getRegistryContract2: Address of Escrow contract is 0x107d9410739e1F557BAD3b0CfA955833070aBaAe
    getRegistryContract2: Address of Exchange contract is 0x95b95256D906664c5047d50Fd2F34cAb741CB44D
    getRegistryContract2: Address of GasPriceMinimum contract is 0x63362e42BF791871ab570AD60B4228C02d9a68F3
    getRegistryContract2: Address of StableToken contract is 0x4917775b54738BFf899C91f320d260Be413f1d16
    getRegistryContract2: Address of GoldToken contract is 0xbe7E2bf1d46f7F351a1785e707Eae8f252Af94Dd
```

### Related issues

- Fixes #302

yerdua committed:
add Escrow and StableToken to the Registry

ashishb commented:
Deploy a new testnet with these changes. Test using https://github.com/celo-org/celo-monorepo/pull/287/ to confirm that your deployed testnet has the issue fixed.

yerdua committed:
Merge branch 'master' into yerdua/register-escrow-and-stable-token-addresses

yerdua committed:
Merge branch 'master' into yerdua/register-escrow-and-stable-token-addresses

yerdua committed:
Merge branch 'master' into yerdua/register-escrow-and-stable-token-addresses

yerdua committed:
Add StableToken and Escrow addresses to the registry before ownership
is tranferred to governance. Since now I know earlier migrations are
editable (pre-mainnet), I've moved the registration of these contracts
into the migrations in which they are deployed.

yerdua committed:
Merge branch 'master' into yerdua/register-escrow-and-stable-token-addresses

yerdua committed:
Merge branch 'master' into yerdua/register-escrow-and-stable-token-addresses

yerdua committed:
Merge branch 'master' into yerdua/register-escrow-and-stable-token-addresses

yerdua committed:
Merge branch 'master' into yerdua/register-escrow-and-stable-token-addresses

yerdua committed:
Alphabetize things. Remove a ".only" from a test file

ashishb committed:
Merge branch 'master' into yerdua/register-escrow-and-stable-token-addresses

yerdua committed:
Merge branch 'master' into yerdua/register-escrow-and-stable-token-addresses

yerdua committed:
Merge branch 'master' into yerdua/register-escrow-and-stable-token-addresses
